internal/printer/container.go

```diff
@@ -122,6 +128,14 @@ func (cc *ContainerConfiguration) Create() (*component.Summary, error) {
 		sections.AddText("Image ID", containerStatus.ImageID)
 	}
 
+	manifest, configuration, err := ManifestManager.GetImageManifest(cc.context, hostOS, c.Image)
+	if err == nil {
+		sections.Add("Image Manifest", component.NewJSONEditor(manifest, true))
+		sections.Add("Image Configuration", component.NewJSONEditor(configuration, true))
+	} else {
+		sections.Add("Image Manifest", component.NewText(fmt.Sprintf("Unable to load image manifest %s", err)))
```

Comment:
The error shown in the UI is hard to read when all the errors from the error chain are concatenated together.
An easy way to reproduce this is to kill the internet before caching and see the error in the pod summary.
It shows something like:
```Unable to load image manifest error creating image source for image docker://nginx:1.14.2: pinging container registry registry-1.docker.io: Get "https://registry-1.docker.io/v2/": dial tcp: lookup registry-1.docker.io: no such host```.
The error from https://github.com/vmware-tanzu/octant/pull/2695/files#diff-85fedc5e144e89139dc0296c0fb2574d2576d493b8bad648bd79c31d32dbd73bR63 and others in the manifest.go and any subsequent errors can be formatted a little better with some delimiters.

Reply:
This change is not a huge blocker and can be made afterwards.